src/styles/hc-2018/ternary.styl

```diff
@@ -201,6 +210,12 @@ a::after {
 a[rel="external"]::after {
     content:"";
 }
+a[rel="prev"]::after {
```

Comment:
Do you when we're displaying a backwards arrow it makes more sense to do it before the text?  There is the issue of other text moving on hover, but this is the case with having it after too.

Reply:
This is stylistic choice in the design community.
I think both choices are equally valid: the Massimo Vignelli went one way ![MTA Manual](https://s3.amazonaws.com/standardsmanualimages/gallery/original_062-2500.jpg) and the DRU went the other. ![British Rail Manual](http://www.doublearrow.co.uk/manual/3_14.1965-05.gif) Both of these are extremely famous pieces of graphic design.
I happen to like the Vignelli solution, as it makes the system a lot more predictable 😊